src/libs/antares/study/parameters.cpp

```diff
@@ -989,6 +977,12 @@ static bool SGDIntLoadFamily_Legacy(Parameters& d,
 
     if (key == "shedding-strategy-global") // ignored since 4.0
         return true;
+
+    if (key == "shedding-strategy") // Was never used
```

Comment:
Keep the same format
if (key == "shedding-strategy") // Was never used
    return true;